recipes/libpng/all/conanfile.py

```diff
@@ -150,6 +151,8 @@ def _configure_cmake(self):
             self._cmake.definitions["PNG_INTEL_SSE"] = self._neon_msa_sse_vsx_mapping[str(self.options.sse)]
         if self._has_vsx_support:
             self._cmake.definitions["PNG_POWERPC_VSX"] = self._neon_msa_sse_vsx_mapping[str(self.options.vsx)]
+        if is_apple_os(self) and self.settings.os != "Macos":
+            self._cmake.definitions["SKIP_INSTALL_PROGRAMS"] = True
```

Comment:
```suggestion
        # Fix iOS/tvOS/watchOS shared
        self._cmake.definitions["CMAKE_MACOSX_BUNDLE"] = False
```

Reply:
@SpaceIm thanks! works as well.

Do you agree to set `CMAKE_MACOSX_BUNDLE` unconditionally? CMake uses it only on Apple platforms anyway.

Reply:
Yes